### PR TITLE
chore(CI): Update master php testing versions and workflow templates

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: .github/workflows/phpunit-pgsql.yml [hunk 1]
```diff
@@ -97,36 +97,45 @@ jobs:
         run: |
           mkdir data
           ./occ maintenance:install --verbose --database=pgsql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass password
-          ./occ app:enable ${{ env.APP_NAME }}
+          ./occ app:enable --force ${{ env.APP_NAME }}
```
Comment (ChristophWurst) [inline .github/workflows/phpunit-pgsql.yml:100]: :smiling_face_with_tear:

File: composer.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
 		"classmap-authoritative": true,
 		"autoloader-suffix": "SuspiciousLogin",
 		"platform": {
-			"php": "7.4"
+			"php": "8.0"
```
Comment (ChristophWurst) [inline composer.json:42]: sneaky sneaky. this implicitly makes Dependabot bump to 7.4 incompatible dependencies. let's do this together with the update of the min php in info.xml

File: .github/workflows/phpunit-pgsql.yml (hunk 1 above)
Comment (st3iny) [reply .github/workflows/phpunit-pgsql.yml:100]: https://github.com/nextcloud/suspicious_login/pull/806

File: composer.json (hunk 2 above)
Comment (st3iny) [reply composer.json:42]: https://github.com/nextcloud/suspicious_login/issues/807

**Comment by st3iny** [comment]
Superseded by
- #802
- #806
- #807

**Pull request closed without merging**